NAPS2.Lib/ImportExport/ExportController.cs

```diff
@@ -1,9 +1,14 @@
-﻿using NAPS2.EtoForms;
+﻿using Eto.Forms;
+using NAPS2.EtoForms;
 using NAPS2.EtoForms.Notifications;
 using NAPS2.EtoForms.Ui;
 using NAPS2.ImportExport.Email;
 using NAPS2.ImportExport.Images;
 using NAPS2.Pdf;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Text;
+using Eto.Forms;
```

Comment:
[Nitpick](https://curtiseinsmann.medium.com/how-to-nitpick-on-code-reviews-with-empathy-d6cb8e670ca4): Dieser Import kann raus, davor warnt auch unsere CI. Denn `Eto.Forms` wird weiter oben bereits importiert.